phpstan.neon.dist

```diff
@@ -17,7 +17,7 @@ parameters:
         - '~^Call to an undefined method PDO::sqliteCreateFunction\(\)\.\z~'
 
         # https://github.com/phpstan/phpstan/issues/1847
-        - '~^Parameter #2 \$registeredAliases of static method Doctrine\\DBAL\\Query\\QueryException::nonUniqueAlias\(\) expects array<string>, array<int, int|string> given\.\z~'
+        - '~^Parameter #2 \$registeredAliases of static method Doctrine\\DBAL\\Query\\QueryException::nonUniqueAlias\(\) expects array<string>, array<int, int\|string> given\.\z~'
```

Comment:
Now it should be `Doctrine\DBAL\Query\Exception\NonUniqueAlias::new()` instead of `Doctrine\DBAL\Query\QueryException::nonUniqueAlias()` until the PHPStan issue is fixed (please update the link to the newly created issue).

Reply:
Done in 8272dea2d6f6a7274593094521da50bab0b4de49.